client/src/ImageSelector.jsx

```diff
@@ -8,6 +8,7 @@ import {
 } from './utils'
 import './style.css'
 import { debounce } from 'lodash'
+import 'itk-vtk-viewer'
```

Comment:
-> something like: 

```
import createViewer from 'itk-vtk-viewer/src/createViewer'
```

The idea: we do not use the exports of the package (which is pointing to a bundled version). Avoid the current difficulties bundling the bundle. In the future we will have nice ESM to work with. For now, we will also have to set up the WebPack rules similar to itk-vtk-viewer: 1) the `resolve` `alias` for `itkConfig.js`. 2) The vtk.js WebPack rules for building the shades, CSS, etc.

CC @PaulHax